transformato/testing.py

```diff
@@ -0,0 +1,111 @@
+conf = '/home/master/transformato/config/ethane-methanol-solvation-free-energy.yaml'
+configuration = load_config_yaml(config=conf,
+                                 input_dir='/home/master/transformato-systems', 
+                                 output_dir='/home/master/Master/test') #user_input 
+
+for i in steps_map:
+    configuration['simulation']['parameters']['nstep'] = i[0]
+    configuration['simulation']['parameters']['nstdcd'] = i[1]
+
+    # load systems
+    s1 = SystemStructure(configuration, 'structure1')
+    s2 = SystemStructure(configuration, 'structure2')
+    a = ProposeMutationRoute(s1, s2)
+    
+    #generate mutation_map
+    guess = [2,4,6]
+    mutation_list1 = []
+    mutation_list2 = []
+    for i in guess:
+        tmp1 = a.generate_mutations_to_common_core_for_mol1(nr_of_steps_for_el=i, nr_of_steps_for_bonded_parameters=i)
+        mutation_list1.append(tmp1)#list of lists
+        tmp2 = a.generate_mutations_to_common_core_for_mol2(nr_of_steps_for_el=i)
+        mutation_list2.append(tmp2) #list of lists
+        
+    
+    mutation_map = [(x,y) for x in mutation_list1 for y in mutation_list2] #tuple of two lists
+    
+    
+    for j in mutation_map:
+        # write intermediate states for systems
+        i = IntermediateStateFactory(system=s1, mutation_list=j[0], configuration=configuration)
+        i.generate_intermediate_states()
+        paths = pathlib.Path(i.path).glob('**/*.sh')
+        for path in sorted(paths):
+            run_dir = path.parent
+            print(f"Start sampling for: {path}")
+            print(f"In directory: {run_dir}")
+            try:
+                exe = subprocess.run(['bash', str(path), str(run_dir)], check=True, capture_output=True, text=True)
+            except TypeError:
+                exe = subprocess.run(['bash', str(path), str(run_dir)], check=True,
+                                     stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            #print(exe.stdout)
+            print('Capture stderr')
+            print(exe.stderr)
+            
+        
+        # write intermediate states
+        i = IntermediateStateFactory(system=s2, mutation_list=j[1], configuration=configuration)
+        i.generate_intermediate_states()
+
+        paths = pathlib.Path(i.path).glob('**/*.sh')
+        for path in sorted(paths):
+            run_dir = path.parent
+            print(f"Start sampling for: {path}")
+            print(f"In directory: {run_dir}")
+            try:
+                exe = subprocess.run(['bash', str(path), str(run_dir)], check=True, capture_output=True, text=True)
+            except TypeError:
+                exe = subprocess.run(['bash', str(path), str(run_dir)], check=True,
+                                     stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            #print(exe.stdout)
+            print('Capture stderr')
+            print(exe.stderr)
+            
+    
+    f = FreeEnergyCalculator(configuration, 'ethane')
```

Comment:
oh, just realizing that `name` variable I suggested in the begging should be a composite variable, like 
`f"{system1}-{system2}'` so that here we can just write:
 `f = FreeEnergyCalculator(configuration, system1)`